src/core/src/package_managers/YumPackageManager.py

```diff
@@ -327,6 +327,10 @@ def get_package_size(self, output):
     # endregion
 
     # region auto OS updates
+    def get_current_auto_os_patch_state(self):
+        """ Gets the current auto OS update patch state on the machine """
+        pass
```

Comment:
Is this a "pass" because it hasn't been implemented yet or because there is nothing to do? A comment explaining why would be useful here and in the other package manager files that define pass for the methods of the interface.

Reply:
+1; A comment is necessary here as it's a short-term thing, and not a legitimate reason for a permanent pass.

The practice is to raise NotImplementedException() in all these cases. The counterpoint is the code cannot be deployed with those exceptions in place - while that's exactly by design, and in many ways desirable, it reduces agility in this case.

Reply:
Pass in this case is because it isn't implemented yet. Added a note for the same